test/cases/ostree-ng.sh

```diff
@@ -562,6 +577,21 @@ for LOOP_COUNTER in $(seq 0 30); do
     sleep 10
 done
 
+# Reboot one more time to make /sysroot as RO by new ostree-libs-2022.6-3.el9.x86_64
```

Comment:
The package NVR is specific to el9, but the same code will be executed also on Fedora. So using just the relevant version and not the fill NVR would be probably better.

In addition to that, could you please explain (e.g. in the commit message) why an additional reboot is need at all? What are you trying to achieve / test, which is not possible to do without this reboot?

Reply:
What's NVR? I suppose it is el9.x86_64? I will remove this, and only use ostree-libs-2022.6-3, please let me know if I am wrong.

And for the reboot thing, with new ostree package, the edge vm needs to reboot to make the /sysroot readonly on some OS. The reboot is not required on all OS, for example, rhel9.2. But instead of adding more if/else to check the os, I reboot the vm for all os, it's quick and will keep the test script simple.
Details can be found in https://docs.google.com/document/d/15fgxZGiQUcEVgefb6s3mP3jQaIBwal9KQIuajK_znU4/edit#  if you are interested.

Reply:
Sorry, "NVR" stands for "name-version-release" when referring to package names. It is a commonly used abbreviation, so I didn't realize that you may not be familiar with it. Your deduction is correct, that only the "release" part is not relevant for other distributions.

Thanks for the explanation of the reboot. I would appreciate if this info was added to the comment in the script or at least to the commit message (without the link to the internal document). The reason why I'm asking for it is that the reason is no obvious just from reading the test case script.

Reply:
Thank you!
Updated the comment.